draft-ietf-httpbis-priority.md

```diff
@@ -646,7 +672,7 @@ Frame Type:
 : PRIORITY_UPDATE
 
 Code:
-: 0xF
+: 0xF and 0x10
```

Comment:
I would skip a number here and let people use the low bit of the type as a flag if they want.  Otherwise, this is a tiny bit more awkward.